### PR TITLE
loading screen active if quote author is empty

### PR DESCRIPTION
**Pull request by zotkin-vyacheslav**
maybe it will solve #https://github.com/Sispheor/MMM-quote-of-the-day/issues/13

### CODE REVIEW

File: MMM-quote-of-the-day.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ Module.register('MMM-quote-of-the-day', {
         quoteTextDiv.className = "normal";
         quoteAuthorDiv.className = "small dimmed";
 
-        if (this.result.quoteText && this.result.quoteAuthor){
+        if (this.result.quoteText){
             quoteTextDiv.innerHTML = this.result.quoteText;
             quoteAuthorDiv.innerHTML = this.result.quoteAuthor;
```
Comment (Sispheor) [inline MMM-quote-of-the-day.js:41]: so maybe add a test here to add it only if quoteAuthor not null?

Comment (zotkin-vyacheslav) [reply MMM-quote-of-the-day.js:41]: quote can be provided without an author. in such a case, we must only display the quote and empty author

Comment (Sispheor) [reply MMM-quote-of-the-day.js:41]: But the line `quoteAuthorDiv.innerHTML = this.result.quoteAuthor;` will not fail here if quoteAuthor not in "result?
That's why I say it should be tested first

Comment (zotkin-vyacheslav) [reply MMM-quote-of-the-day.js:41]: > will not fail here if quoteAuthor not in "result?

no, it won't fall, author just will be empty

Comment (Sispheor) [reply MMM-quote-of-the-day.js:41]: Ok then
